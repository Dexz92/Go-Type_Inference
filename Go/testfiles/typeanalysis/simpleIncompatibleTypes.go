func main() {
  var i int = 2;
  var b bool = i;
}
