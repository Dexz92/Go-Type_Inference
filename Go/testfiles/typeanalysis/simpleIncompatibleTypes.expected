Error at line 3: Trying to assign int to bool variable
